app/controllers/profiles_controller.rb

```diff
@@ -10,7 +9,7 @@ def edit
 
   def update
     if current_user.update(user_params)
-      redirect_to profile_path, flash: { warning: "User was successfully updated." }
+      redirect_to profile_path, flash: { warning: t('user.updated') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.